uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/FeedScreen.kt

```diff
@@ -372,10 +329,21 @@ private fun FeedList(
                     )
                 }
             }
-            isListFinished.value = listState.firstVisibleItemIndex + listState.layoutInfo
-                .visibleItemsInfo.size == listState.layoutInfo.totalItemsCount
             if (listState.firstVisibleItemIndex != 0) {
                 item {
+                    val isListFinished by remember {
```

Comment:
I moved this process with https://github.com/DroidKaigi/conference-app-2021/pull/406/commits/efccce872942b562ade60fe507ab89039a5c2ac5 .

Because LazyColumn was recomposed unnecessarily many times.